python/lsst/analysis/tools/actions/plot/focalPlanePlot.py

```diff
@@ -278,6 +278,15 @@ class FocalPlaneGeometryPlot(FocalPlanePlot):
         },
     )
 
+    def getInputSchema(self, **kwargs) -> KeyedDataSchema:
+        base = []
+        base.append(("detector", Vector))
+        if self.level == "amplifier":
```

Comment:
Can level ever be anything else? What is the default? Where is it defined? If we only forsee it being amplifier or nothing then maybe a True/False config option would be better.

Reply:
`level` can be either "amplifier" or "detector". The default is "amplifier", but if it is set to "detector", then the user shouldn't need to load the "amplifier" vectorKey. It's defined on line 272 in the same file.

Reply:
Sorry, for some reason I thought this was a new option added on this ticket. Opps.